scripts/cnv_wdl/somatic/cnv_oncotate.wdl

```diff
@@ -0,0 +1,44 @@
+workflow CNVOncotateCalledSegments {
+    File called_file
+    Int? mem
+    String? oncotator_docker
+    String? oncotator_output_type
+    String? additional_args
+
+    call OncotateCnvSegments {
+        input:
+            called_file=called_file,
+            mem=mem,
+            oncotator_docker=oncotator_docker,
+            oncotator_output_type=oncotator_output_type,
+            additional_args=additional_args
+    }
+
+    output {
+        File oncotated_called_file = OncotateCnvSegments.oncotated_called_file
+    }
+}
+
+task OncotateCnvSegments {
```

Comment:
Use of camel case here is inconsistent with above.  I think the task can just be named `OncotateSegments`.